### DISCUSSION TITLE
Virustotal and Malwarebytes flagging systembridge.exe as trojan

### DISCUSSION THREAD

**Discussion by james2maybe** [timmo001/system-bridge#2869] [General]
I'm guessing these are false positives?

![Screenshot_1](https://github.com/timmo001/system-bridge/assets/38017505/a7be915a-61d7-45a8-b57f-cbf3cf1fafe5)

**Comment by Clusters** [comment]
2 years later, unfortunately, the package is still detected as trojan

I wanted to add system-bridge to the Winget package manager. But with the fact that this is detected as a trojan by Microsoft, I rather don't do that, regardless of it being a false positive detection.

<img width="954" height="662" alt="image" src="https://github.com/user-attachments/assets/66270edf-d3da-43b1-a0d6-710a659a64d9" />

**Comment by robvdl** [comment]
The issue appears to be that it's flagging the installer nullsoft install system.

What if the Windows application changed to a different installer?

**Reply by robvdl** [reply]
But Nullsoft installer is now Open Source so I see why it was used. Back in my day as a Delphi programmer several decades ago in a previous life, I would use InnoSetup but I wouldn't recommend it today anymore. I liked InnoSetup but it's really old tech now.
